UnitsNet/UnitDefinitions/FoulingFactor.json

```diff
@@ -0,0 +1,68 @@
+{
+  "Name": "FoulingFactor",
+  "BaseUnit": "HourSquareFeetDegreefahrenheitPerBtu",
+  "XmlDoc": "Heat Transfer Coefficient or Thermal conductivity - indicates a materials ability to conduct heat.",
+  "Units": [
+    {
+      "SingularName": "HourSquareFeetDegreefahrenheitPerBtu",
+      "PluralName": "HourSquareFeetDegreefahrenheitsPerBtu",
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "hr*ft2�F/Btu" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "SquareMeterDegreeCentigradePerWatt",
+      "PluralName": "SquareMeterDegreeCentigradesPerWatt",
+      "FromUnitToBaseFunc": "x/5.6787",
+      "FromBaseToUnitFunc": "x*5.6787",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "m2�C/W" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "SquareCentimeterKelvinPerWatt",
+      "PluralName": "SquareCentimeterKelvinsPerWatt",
+      "FromUnitToBaseFunc": "x/0.0005678",
+      "FromBaseToUnitFunc": "x*0.0005678",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "cm2K/W" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "SquareCentimeterHourDegreeCentigradePerKilloCalorie",
```

Comment:
Here the time unit `Hour` is after the area unit `SquareCentimeter`, but the unit `HourSquareFeetDegreefahrenheitPerBtu` above has the time unit before the area unit. That seems inconsistent to me, which is correct or preferred?